I/O boost docs (#3610)
timescale/docs: use-timescale/services/i-o-boost.md

> ---

Reviewer: @esclymeTS ,  I have had a bit of a think about this. Would it be best to add this page to the https://docs.timescale.com/use-timescale/latest/data-tiering/ section and add links to it from the main page.

title: I/O boost
excerpt: Increase I/O and throughput to avoid performance bottlenecks and enhance scalability
products: [cloud]
keywords: [io, io boost, performance]
---

# I/O boost

> I/O boost allows you to increase I/O and throughput of a service's EBS storage on the need-only basis. This means you can enable it for the most demanding applications, while keeping costs under control. 

Reviewer: ```suggestion
You use I/O boost to increase I/O and throughput of a service's high-performance storage on the need-only basis. This means you can enable it for the most demanding applications, while keeping costs under control. 
```
Add a link to https://docs.timescale.com/use-timescale/latest/data-tiering/.

> You use I/O boost to increase I/O and throughput of a service's [high-performance storage][data-tiering] on the need-only basis. This means you can enable it for the most demanding applications, while keeping costs under control.

Reviewer: ```suggestion
You use I/O boost to increase I/O and throughput of a service's [high-performance storage][data-tiering]. This means you can enable it for the most demanding applications, while keeping costs under control.
```

> Enabling I/O boost increases I/O to 16,000 IOPS and throughput to 1,000 MBps. The boost also applies to any [high-availability][ha-replicas] or [read replicas][read-replicas] you might have running for a service, although for an additional fee. 

Reviewer: ```suggestion
Enabling I/O boost increases I/O to 16,000 IOPS and throughput to 1,000 MBps. The boost also applies to any [high-availability][ha-replicas] replicas you might have running for a service, although for an additional fee. 
```
